src/engraving/dom/cmd.cpp

```diff
@@ -2157,10 +2157,51 @@ bool Score::toggleArticulation(EngravingItem* el, Articulation* a)
         undoRemoveElement(oa);
         return false;
     }
+
+    if (!a->isDouble()) {
+        addArticulationToChord(a, c);
+        return true;
+    }
+
+    // Split the new articulation into "sub-components" and collect the unique ones (not already present in chord)...
+    std::set<Articulation*> uniqueSubComponents;
+
+    std::set<SymId> newSubComponentIds = splitArticulations({ a->symId() });
+    for (const SymId& id : newSubComponentIds) {
+        Articulation* articCopy = Factory::createArticulation(score()->dummy()->chord());
+        articCopy->setSymId(id);
+
+        if (!c->hasArticulation(articCopy)) {
+            uniqueSubComponents.emplace(articCopy);
+            continue;
+        }
+        delete articCopy;
+    }
+    // If the new articulation is not fully unique, only add the necessary sub-components (prevents duplication)...
+    if (newSubComponentIds.size() > uniqueSubComponents.size()) {
+        for (Articulation* newArtic : uniqueSubComponents) {
+            addArticulationToChord(newArtic, c);
+        }
+        return true;
+    }
+    // If the new articulation is fully unique, the original can be added...
+    for (Articulation* subComponent : uniqueSubComponents) {
+        delete subComponent;
+    }
+
+    addArticulationToChord(a, c);
```

Comment:
If I'm not mistaken, it's never desired anymore to add a compound articulation symbol to a chord; instead, the symbols should always be added as separate Articulation items. @oktophonie Please correct me if I'm wrong.

There is some inconsistency about this:
1. Start note input mode, and in the note input toolbar, select the "staccato" and "marcato" buttons.
 -> when adding a new note, this produces two Articulation items, one for each symbol (as expected, as far as I know)
2. Exit note input mode, and select the added note. 
3. With the note selected, enter note input mode again. The articulations of this note are selected in the note input toolbar.
4. Enter another note. 
 -> This note gets one Articulation item with the compound symbol. (not as expected, as far as I know)

I think we should try to keep the code for adding articulations a bit more streamlined. Now there are a lot of methods related to it:
- `Chord::drop` (well, that's fine, because `drop` is a widely used method)
- `Score::toggleArticulation`
- `Score::addArticulationToChord` (which contains logic that seems to duplicate logic from `Chord::add`)
- `Chord::updateArticulations`
- `Chord::add`
- some toplevel functions like `updateArticulations` in articulation.h/cpp
- and doubtlessly more

This makes it difficult to know which method to use for which purpose. I hope we can do with just a few methods.

Reply:
> instead, the symbols should always be added as separate Articulation items.

Personally very much on board with this. If you want to get a feel for this @oktophonie, the latest push to this PR should prevent the inconsistency described above.

Reply:
@cbjeukendrup It's certainly the intention that, when using the toolbar, the compound symbols are never used. But they still exist in SMuFL, and are in the palette (though we should hide them away under 'More'), and it seemed a bit much to absolutely prevent them being used if the user has a specific reason to want them, but applying those compound symbols directly from the palette should be the only way it should be achievable.

We can rethink that if it's way more trouble than it's worth.